src/containers/Home/index.jsx

```diff
@@ -1,27 +1,44 @@
 import React, { Component } from 'react';
+import classnames from 'classnames';
 import { observer, inject } from 'mobx-react';
 
+import { getScrollTop } from 'src/utils';
+
 import Nav from 'components/Nav';
+import Banner from 'components/Banner';
 import AppList from 'components/AppList';
 import styles from './index.scss';
 
-@inject(({ rootStore }) => ({
-  appStore: rootStore.appStore,
-  config: rootStore.config,
-}))
+@inject('rootStore')
 @observer
 export default class Home extends Component {
   static async onEnter({ appStore }) {
     await appStore.fetchApps();
   }
 
+  componentDidMount() {
+    window.onscroll = this.handleScoll;
+  }
+
+  handleScoll = () => {
+    const store = this.props.rootStore;
+    if (!store.fixNav && getScrollTop() > 345) {
+      store.fixNav = true;
+    } else if (store.fixNav && getScrollTop() < 345) {
+      store.fixNav = false;
```

Comment:
`props` can be modified ?

Reply:
em, i'd better to use mobx action to modify state